app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,32 @@
+<h2>Sign up</h2>
+
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name), html: { data: { turbo: false} }) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+ <div class="field">
+    <%= f.label :name %><br />
+    <%= f.text_field :name, autofocus: true, autocomplete: "name" %>
+  </div>
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %>
+    <% if @minimum_password_length %>
+    <em>(<%= @minimum_password_length %> characters minimum)</em>
+    <% end %><br />
+    <%= f.password_field :password, autocomplete: "new-password" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password_confirmation %><br />
+    <%= f.password_field :password_confirmation, autocomplete: "new-password" %>
+  </div>
+
+  <div class="actions">
+    <%= f.submit "Sign up" %>
+  </div>
+<% end %>
+
+<%= render "devise/shared/links" %>
```

Comment:
- [ ] [REQUIRED] You have really done a great job @BouaikIlham adding devise to your blog project, I was able to sign up successfully, and create posts and comments. However, you need to ensure the design follows the provided wireframe as stated in the project requirement.
- [ ] Kindly ensure that likes are enabled on the post and comments as required in the project speculations.

Take a look at this for the specified [wireframe](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/sneak_peek.md).

### Here is a picture of what your Post, Comment Creation looks like
![Screenshot from 2022-09-08 13-25-55](https://user-images.githubusercontent.com/58692590/189122269-60e2c730-4304-4917-a48b-3e10d1ee9b8a.png)
![Screenshot from 2022-09-08 13-25-49](https://user-images.githubusercontent.com/58692590/189122327-ed039231-3129-4040-9ac7-526441ebfc03.png)
![Screenshot from 2022-09-08 13-25-34](https://user-images.githubusercontent.com/58692590/189122352-96062e98-81f8-443d-aa96-5e9c296c839c.png)